docs/source/release-notes.rst

```diff
@@ -3,17 +3,72 @@ FiftyOne Release Notes
 
 .. default-role:: code
 
-.. _release-notes-v0.23.7:
+FiftyOne Teams 1.5.8
+--------------------
+*Released March 21, 2024*
+
+Includes all updates from :ref:`FiftyOne 0.23.7 <release-notes-v0.23.7>`.
 
 FiftyOne 0.23.7
 ---------------
-*Released March 8, 2024*
+*Released March 21, 2024*
 
 App
 
-* Updated `Have a Team?` link with
+- Updated `Have a Team?` link with
   `Book a demo <https://voxel51.com/book-a-demo/?utm_source=FiftyOneApp>`_
   page
+- Fixed indexed BooleanFields in Lightning Mode
+  `#4139 <https://github.com/voxel51/fiftyone/pull/4139>`_
+- Fixed app crash when many None-valued fields exist in the sample modal
+  `#4154 <https://github.com/voxel51/fiftyone/pull/4154>`_
+
+Docs
+
+- Added an `Albumentations <https://albumentations.ai/docs/>`_ image augmentation :ref:`integration overview <albumentations-integration>`
+  `#4155 <https://github.com/voxel51/fiftyone/pull/4155>`_
+- Adding Places dataset to the zoo
```

Comment:
```suggestion
- Added `Places dataset <http://places2.csail.mit.edu/index.html>`_ to the zoo
```

Reply:
note this link is currently broken so I don't know what's up but it is the correct link I believe